ui/src/pages/frontsite/components/LiteratureRelations/LiteratureRelations.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List, Divider } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import { FrontSiteRoutes } from '@routes/urls';
+import { SeparatedList } from '@components';
+import { DocumentEdition } from '@components/Document';
+import _get from 'lodash/get';
+import _isEmpty from 'lodash/isEmpty';
+
+export class LiteratureRelations extends Component {
+  constructor(props) {
+    super(props);
+    this.relations = props.relations;
+  }
+
+  getLinkTo = relation => {
+    return relation.pid_type === 'docid'
```

Comment:
I wanted to use it, but unfortunately that one relies on the existence of the schema field.
That's why Niklas proposed to add the `pid_type` in the records too... to avoid this. If we decide to do it...

Reply:
let's leave it for now, it is already a lot of work here. we can keep it in mind for when we will have everywhere the same `pid` object